src/main/java/com/example/DataService/CurrencyService.java

```diff
@@ -0,0 +1,36 @@
+package com.example.DataService;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.example.repository.CurrencyRepository;
+
+import model.Asset;
+import model.Currency;
+
+@Service
+public class CurrencyService {
+	
+	private CurrencyRepository cr;
+	
+	public CurrencyService(CurrencyRepository cr) {
+		this.cr = cr;
+	}
+	
+	public void putCurrencies(){
+
+		final String uri = "https://api.coincap.io/v2/assets/";
```

Comment:
Add this to application properites - think about a situation when an url is changed